src/training/final.ts

```diff
@@ -27,13 +27,82 @@ type Coupon = {
   amount: number;
 };
 
+const coupon_100: Coupon = { couponId: 1, couponName: "coupon_100",amount: 100 };
+const coupon_500: Coupon = { couponId: 2, couponName: "coupon_500", amount: 500 };
+const coupon_1000: Coupon = { couponId: 3, couponName: "coupon_1000", amount: 1000 };
+
 type Wallet = {
-  walletId;
-  username;
-  balance;
-  method;
-  transactionHistroy;
-  coupon;
+  walletId: number;
+  username: string;
+  balance: number;
+  method: 0 | 1 | 2;
+  transactionHistroy: Transaction[];
```

Comment:
すみません。これ僕のミスかもしれないんですけど
```suggestion
  transactionHistory: Transaction[];
```

Reply:
該当プロパティ名を修正しました。